zachang/HelloBooks
# #149915920 user registration

What does this PR do?
* It allows a user create an account on the system

Description of Task to be completed?
* A page is created where the user can create an account on the system.

How should this be manually tested?
+ A user should navigate to the registration page to create an account.

Any background context you want to provide?
+ none for now.

What are the relevant pivotal tracker stories?
`#149915920`

Questions:
+ none for now
